apps/src/code-studio/components/progress/SendLessonDialog.jsx

```diff
@@ -0,0 +1,97 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import Button from '@cdo/apps/templates/Button';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+
+export default class SendLessonDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    handleClose: PropTypes.func.isRequired,
+    showGoogleButton: PropTypes.bool
+  };
+
+  render() {
```

Comment:
Not required, but convention: The render function is usually the last one in the class.